gradle.properties

```diff
@@ -0,0 +1 @@
+org.gradle.jvmargs=-Xmx2g -XX:MaxPermSize=512m
```

Comment:
Setting `-XX:MaxPermSize` will fail on Java 9. Currently, this is not a problem, since no Gradle build is running on Java 9. This could change in the near future.
Would you have a suggestion how to address this problem?